workflow/rules/annotation/pass1.smk

```diff
@@ -1,6 +1,18 @@
+rule download_snpeff_db:
+    output:
+        directory("{working_dir}/results/databases/{{ref}}".format(working_dir=os.getcwd()))
+    params:
+        db_dir=os.path.join(os.getcwd(), "results/databases/")
```

Comment:
results/databases is not a portable parameter, because the output path can be altered by flags like --default-remote-prefix. You have to use a function here instead: `lambda _, output: os.path.dirname(output[0])`.